lucas_scheinkerman/README.md

```diff
@@ -1 +1,45 @@
-# Lucas Scheinkerman
+## Differential-drive robots
+
+A differential-drive robot (DDR from now) is a robot which has a two-wheel mechanism for allowing it to move. There is a third wheel which prevents the robot from falling and helps it balance, but which has not active role in the movement of the robot. br>
+This mechanism for moving the robot is based on the difference between the two wheels angular velocities, which is why it is called _differential_-drive.
+
+Before starting to describe how it works, it's important to notice that there are three velocities involved: The two velocities that correspond to the wheels and the velocity of the robot.
+
+-The moving direction in a two-dimensional plane can be described by two parameters/ways of movement: **Traslation** and **rotation**. Both of which we'll define realtive to the center of the robot. <br>
+We'll start by explaining the simple cases: when each of the parameters appears alone. That is, without the presence of the other parameter.
+**Pure traslation** in the DDR occurs only when the two wheels angular velocities have the same module and sign. Similarly, **pure rotation** occurs in the DDR only when the two wheels have the same angular-velocity module but different sign. <br>
+In the rest of the cases, both parameters are present together. So, in an intuitive way, we can see that when the angular-velocities will have the same sign but different modules, the robot will move in the same direction as the direction of the angular velocities sign, but with a rotational movement, which becomes more predominant as the difference between the modules grows. The extreme case of this scenario is when one of the wheels has no angular velocity at all, in which case the robot rotates along the Z-axis that passes through the wheel that has no velocity (supposing that the plane in which the DDR moves is in the X-Y plane). That is, it has rotation and traslation with our definition of the reference as the center of the robot, but pure rotation if we define the reference in the zero-velocity wheel.
+
+
+-Besides the angular velocity of the wheels, there are two more factors that define the pose variation (position and direction) of the robot. These are the **wheels radius** and the **distance between the two wheels**. <br>
```

Comment:
You will see this a lot in ROS:

pose = position + orientation

So, use "orientation" instead of "direction".